src/ConnectorRuntimeConfig.ts

```diff
@@ -22,10 +23,6 @@ export interface ConnectorRuntimeConfig extends RuntimeConfig {
     modules: Record<string, ConnectorRuntimeModuleConfiguration>;
 
     infrastructure: {
-        httpServer: {
-            enabled: boolean;
-            apiKey: string;
-            cors?: any;
-        };
+        httpServer: HttpServerConfiguration;
```

Comment:
you have now exposed the port and helmetOptions which we didn't do on purpose. please undo.

Reply:
That is the ConnectorRuntimeConfig and not the connectorConfig that is used for the config file.

In the schema file, the port is already exposed.

Reply:
nvm, I think it's ok.